python-avd/pyavd/_eos_designs/schema/schema_fragments/defs_network_services.schema.yml

```diff
@@ -830,6 +830,11 @@ $defs:
                       items:
                         type: str
                         description: IPv4_address/Mask.
+                    ip_address_secondaries:
+                      type: list
+                      items:
+                        type: str
+                        description: IPv4_address/Mask.
```

Comment:
this will not work well in this model. The model is built to index per device, and adding another list, that will be the same for all, will be weird. I think we need to skip this here, and only add it for port-channels and svis. Alternatively we need a list of lists, which is also terrible to work with.

Reply:
@gmuloc